src/beliefpropagation/beliefpropagation.jl

```diff
@@ -134,30 +112,30 @@ function belief_propagation_iteration(
 end
 
 function belief_propagation_iteration(
-  tn::ITensorNetwork,
-  mts::DataGraph;
-  contract_kwargs=(; alg="density_matrix", output_structure=path_graph_structure, maxdim=1),
+  ptn::PartitionedGraph,
+  mts;
+  contractor=default_contractor,
   compute_norm=false,
-  edges=edge_sequence(mts),
+  edges=PartitionEdge.(edge_sequence(partitioned_graph(ptn))),
```

Comment:
Should separate off into a `default_edge_sequence(::PartitionedGraph)` function.